### ISSUE TITLE
Wrong URLs in Get involved page

### ISSUE THREAD

**Issue by ymhuang0808** [sequelpro/sequelpro#2715]

**Comment by stuconnolly** [comment]
Yep, there's a ton of links within the docs pages which are wrong.

**Comment by josegonzalez** [comment]
@stuconnolly is the website source available somewhere? I'd be happy to fix those up if so :)

**Comment by josegonzalez** [comment]
Probably worth consolidating this with #2620 and tracking broken links in one place :)

**Comment by stuconnolly** [comment]
Consolidating all of the website issues on this one (mainly because it has the most info):

* #2625: Missing screenshot on https://sequelpro.com/docs/ref/core-features/relations
* #2620: Redirect issue on https://sequelpro.com/docs/Install_MySQL_on_your_Mac 

@josegonzalez Afraid the website source isn't on GitHub yet, but we're working on it.
